### PR TITLE
U/danielsf/tiles

### PR DESCRIPTION
**Pull request by danielsf**
This is the pull request in support of moving the CatSim server from the Windows machine over to epyc.  Here are the instructions for actually connecting remotely to the database on epyc (I will slack you the password and username).

The ssh command to establish the tunnel to the server is now:

ssh -N -L 51433:localhost:1433 lsst-sims@epyc.astro.washington.edu

The new CatalogDBObject classes you want to use are:

StarObj -> UWStarCatalogObj
GalaxyTileObj -> UWGalaxyTileObj
GalaxyDiskObj -> UWGalaxyDiskObj
GalaxyBulgeObj -> UWGalaxyBulgeObj
GalaxyAgnObj -> UWGalaxyAgnObj

all of which can be imported from lsst.sims.catUtils.baseCatalogModels

### CODE REVIEW

File: python/lsst/sims/utils/htmModule.py [hunk 1]
```diff
@@ -1152,3 +1268,78 @@ def halfSpaceFromRaDec(ra, dec, radius):
...
+
+    Returns
+    -------
+    A list of the cartesian points where the Half Spaces intersect
+
+    Based on section 3.5 of
+    Szalay A. et al. (2007)
+    "Indexing the Sphere with the Hierarchical Triangular Mesh"
+    arXiv:cs/0701164
+    """
```
Comment (jbkalmbach) [inline python/lsst/sims/utils/htmModule.py]: Should we include a case where the half spaces are the same? Are we assuming the user won't do this? Or am I missing it in here?

Comment (danielsf) [reply python/lsst/sims/utils/htmModule.py]: That case will be encompassed in the "Half spaces are based on parallel planes that don't intersect" case at line ~ 1320

I take your point, though, that returning an empty list might not be the right thing to do there.

Comment (danielsf) [reply python/lsst/sims/utils/htmModule.py]: Would it be sufficient to just add a line to the docstring explaining that the list will be empty if the half spaces are identical?

Comment (jbkalmbach) [reply python/lsst/sims/utils/htmModule.py]: Yeah, that would be fine. Just a statement to help somebody understand what would happen in that case before using it is good.

**Pull request merged**